source/operator.rst

```diff
@@ -111,7 +125,35 @@ configuration options for the Percona Server for MySQL.
 | **Example**     | ``0``                                                                                     |
 +-----------------+-------------------------------------------------------------------------------------------+
 | **Description** | The number of the Percona Server for MySQL `semi-sync                                     |
-|                 | <https://dev.mysql.com/doc/refman/5.7/en/replication-semisync.html>`_ replicas            |
+|                 | <https://dev.mysql.com/doc/refman/8.0/en/replication-semisync.html>`_ replicas            |
++-----------------+-------------------------------------------------------------------------------------------+
+|                                                                                                             |
++-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _mysql-primaryServiceType:                                                             |
```

Comment:
What does it do? Can I configure other service parameters here? Like set annotations for this service? Is it for GR only?

Reply:
thanks! I've added a note that these parameters are for asynchronous replication only